### PR TITLE
drop support for multiple tabs in VideoWorkspace

### LINKED ISSUE #1913: Drop support for multiple tabs in VideoWorkspace
**Context**
Supporting multi tab setting for NFT issuing (inherited from studio)

**Design links**

The video tabs component was removed and replaced by the local component video workspace toolbar - which can be found here : https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=1341%3A133231

All of the screens were adjusted to support new component. It's worth noticing how it behaves on small screens: https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=969%3A17422

⚠️  🆕  Component:

- [ ] https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=5096%3A145926

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #1913 

Design:
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=5096%3A145926
https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=969%3A17422
https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=0%3A1

### CODE REVIEW

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { useRef } from 'react'
+
+import { Pill } from '@/components/Pill'
+import { IconButton } from '@/components/_buttons/IconButton'
+import { SvgActionClose } from '@/components/_icons'
+
+import { ButtonsContainer, Tab, TabTitle, TabWrapper, TabsContainer, Topbar } from './DrawerHeader.styles'
+
+type TabsBarProps = {
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]:
```suggestion
type DrawerHeaderProps = {
```

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  title?: string
+  label: string
+  onCloseClick: () => void
+}
+
+export const DrawerHeader: React.FC<TabsBarProps> = React.memo(({ onCloseClick, title, label }) => {
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]:
```suggestion
export const DrawerHeader: React.FC<DrawerHeaderProps> = React.memo(({ onCloseClick, title, label }) => {
```

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  const tabsContainerRef = useRef<HTMLDivElement>(null)
+  return (
+    <Topbar>
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]: Let's name this differently, maybe `Tabbar`?

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      <TabsContainer ref={tabsContainerRef}>
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]: Do we really need three wrappers here? We have here TabsContainer, TabWrapper and Tab. Just asking out of curiosity. 
Also, I think it should be named TabContainer, not Tab**s**Container(without plural form)

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        <TabWrapper>
+          <Tab>
+            {label && <Pill label={label} size="small" />}
+            <TabTitle variant="t200">{title}</TabTitle>
+          </Tab>
+        </TabWrapper>
+      </TabsContainer>
+      <ButtonsContainer>
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]: Again, do we need a wrapper here? I think we could easily create `StyledIconButton`

**Review by drillprop — CHANGES_REQUESTED** [review]
In terms of functionality works pretty well. Found some small issues!
Also:
image
This feels good in VideoWorkspace!
image

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx (hunk 1 above)
Comment (rafalpawlow) [reply packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]: Not copy paste at all 😬

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import { Tab, TabContainer, TabTitle, Tabbar } from './DrawerHeader.styles'
+
+export type DrawerHeaderProps = {
+  title?: string
+  label: string
+  onCloseClick: () => void
+}
+
+export const DrawerHeader: React.FC<DrawerHeaderProps> = React.memo(({ onCloseClick, title, label }) => {
+  const tabsContainerRef = useRef<HTMLDivElement>(null)
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]:
```suggestion
  const tabContainerRef = useRef<HTMLDivElement>(null)
```

File: packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  return (
+    <Tabbar>
+      <TabContainer ref={tabsContainerRef}>
```
Comment (drillprop) [inline packages/atlas/src/components/DrawerHeader/DrawerHeader.tsx]:
```suggestion
      <TabContainer ref={tabContainerRef}>
```

**Review by drillprop — COMMENTED** [review]
You missed those

**Review by drillprop — APPROVED** [review]

**Comment by eldiegod** [comment]
Looks great with no tabs 🤓

**Comment by kdembler** [comment]
Will take a look at this once we release new sign in flow so `dev` is "clear"

**Review by kdembler — APPROVED** [review]

**Pull request merged**